index.js

```diff
@@ -644,13 +681,37 @@ function getStatusEffectNamesForVersion (supportFeature) {
   }
 }
 
+function getEntityBB (entity) {
+  const w = entity.width / 2
+  const pos = entity.position
+
+  return new AABB(-w, 0, -w, w, entity.height, w).offset(pos.x, pos.y, pos.z)
+}
+
+function getIntersectingEntities (bb, entityList) {
+  const entities = []
+
+  for (let i = 0; i < entityList.length; i++) {
+    const e = entityList[i]
+    if (e.type !== 'object') {
```

Comment:
```suggestion
    if (e.type !== 'object' && e.type !== 'orb') {
```

don't want to be pushed by `experience_orb`

Reply:
A list of entities that the bot should not be pushed by should be there.

Or how about a property inside an entity marking it as pushable? That way people can set if they want their bot to be pushable by only certain entities

Reply:
> Or how about a property inside an entity marking it as pushable? That way people can set if they want their bot to be pushable by only certain entities

https://github.com/PrismarineJS/prismarine-entity/blob/master/index.js#L24
I already committed an attribute for noclip, I guess I'll need to actually set it places 😅 

```suggestion
    if (!e.noClip && (e.type !== 'object' && e.type !== 'orb')) { // temp until I can set noclip flag in places
```